resources/views/components/button/enhanced.blade.php

```diff
@@ -1,3 +1,3 @@
-<x-rapidez-ct::button {{ $attributes->merge(['class' => 'bg-ct-enhanced text-sm font-semibold text-white']) }}>
+<x-rapidez-ct::button {{ $attributes->merge(['class' => 'bg-ct-enhanced text-sm font-semibold text-white border-b border-primary/20']) }}>
```

Comment:
i think you want to create a subtle border bottom, but instead of the text color, can you just use black with an opacity? just in case someone else uses a weird text color